pmm/pmm2-ovf-test.groovy

```diff
@@ -117,29 +129,43 @@ pipeline {
                     
                     tar xvf \$VM_NAME.ova
                     export ovf_name=$(find -type f -name '*.ovf');
+                    ifconfig
+                    ip addr show
+                    sudo yum install -y NetworkManager
```

Comment:
I can install when I start the DO instance, but as you wish.

Reply:
if you can add it when you instantiate, I will be glad.